textacy/datasets/wikipedia.py

```diff
@@ -88,7 +88,8 @@
 re_brackets = re.compile(r'\[\s?\]|\(\s?\)')
 
 re_comments = re.compile('<!--.*?-->', flags=re.UNICODE | re.DOTALL)
-re_categories = re.compile(r'\[\[(%s)[^\]\[]*\]\]' % '|'.join(MAPPING_CAT.values()), flags=re.UNICODE)
+
+re_categories = re.compile(r'\[\[({})[^\]\[]*\]\]'.format('|').join(MAPPING_CAT.values()), flags=re.UNICODE)
```

Comment:
_Although_: Is that not an extra paren before `.join`? I'll hotfix in master if so.

Reply:
yes sorry my bad

Reply:
no worries 👌

Reply:
I was just about to fix it before you merged, you too quick ;)
i'll make sure next time to restart kernel in jupyter 🙈